src/org/jitsi/util/RTPUtils.java

```diff
@@ -294,4 +294,75 @@ public static boolean isNewerTimestampThan(long a, long b)
     {
         return rtpTimestampDiff(a, b) > 0;
     }
+
```

Comment:
Are these used anywhere?

Reply:
i used them as part of debugging, have found them useful and have had to re-add them a couple times now so thought they would be useful to have.  i can remove/move them if you feel strongly about that.

Reply:
No, I like them. I've used something similar more than once, but never thought to commit it...